[2021-07-20 19:49:16,462] {taskinstance.py:876} INFO - Dependencies all met for <TaskInstance: Fuse.fuse7 2021-07-19T00:03:00+00:00 [queued]>
[2021-07-20 19:49:16,473] {taskinstance.py:876} INFO - Dependencies all met for <TaskInstance: Fuse.fuse7 2021-07-19T00:03:00+00:00 [queued]>
[2021-07-20 19:49:16,473] {taskinstance.py:1067} INFO - 
--------------------------------------------------------------------------------
[2021-07-20 19:49:16,473] {taskinstance.py:1068} INFO - Starting attempt 1 of 8
[2021-07-20 19:49:16,473] {taskinstance.py:1069} INFO - 
--------------------------------------------------------------------------------
[2021-07-20 19:49:16,550] {taskinstance.py:1087} INFO - Executing <Task(PythonOperator): fuse7> on 2021-07-19T00:03:00+00:00
[2021-07-20 19:49:16,555] {standard_task_runner.py:52} INFO - Started process 1039033 to run task
[2021-07-20 19:49:16,565] {standard_task_runner.py:76} INFO - Running: ['airflow', 'tasks', 'run', 'Fuse', 'fuse7', '2021-07-19T00:03:00+00:00', '--job-id', '327', '--pool', 'default_pool', '--raw', '--subdir', 'DAGS_FOLDER/dag.py', '--cfg-path', '/tmp/tmpun2yxdb1', '--error-file', '/tmp/tmpdj6bf2jd']
[2021-07-20 19:49:16,566] {standard_task_runner.py:77} INFO - Job 327: Subtask fuse7
[2021-07-20 19:49:16,724] {logging_mixin.py:104} INFO - Running <TaskInstance: Fuse.fuse7 2021-07-19T00:03:00+00:00 [running]> on host deepika-Inspiron-5570
[2021-07-20 19:49:16,950] {taskinstance.py:1280} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_EMAIL=sthadpka93@gmail.com
AIRFLOW_CTX_DAG_OWNER=postgres
AIRFLOW_CTX_DAG_ID=Fuse
AIRFLOW_CTX_TASK_ID=fuse7
AIRFLOW_CTX_EXECUTION_DATE=2021-07-19T00:03:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2021-07-19T00:03:00+00:00
[2021-07-20 20:17:37,960] {python.py:151} INFO - Done. Returned value was: None
[2021-07-20 20:17:38,012] {taskinstance.py:1184} INFO - Marking task as SUCCESS. dag_id=Fuse, task_id=fuse7, execution_date=20210719T000300, start_date=20210720T140416, end_date=20210720T143238
[2021-07-20 20:17:38,134] {taskinstance.py:1245} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2021-07-20 20:17:38,150] {local_task_job.py:151} INFO - Task exited with return code 0
